internal/cli/apis.go

```diff
@@ -445,21 +445,15 @@ func deleteAPICmd(cli *cli) *cobra.Command {
 				}
 			}
 
-			return ansi.Spinner("Deleting API(s)", func() error {
-				var errs []error
-				for _, id := range ids {
-					if id != "" {
-						if _, err := cli.api.ResourceServer.Read(cmd.Context(), id); err != nil {
-							errs = append(errs, fmt.Errorf("failed to delete API with ID %q: %w", id, err))
-							continue
-						}
-
-						if err := cli.api.ResourceServer.Delete(cmd.Context(), id); err != nil {
-							errs = append(errs, fmt.Errorf("failed to delete API with ID %q: %w", id, err))
-						}
-					}
+			return ansi.ProgressBar("Deleting API(s)", ids, func(_ int, id string) error {
+				if _, err := cli.api.ResourceServer.Read(cmd.Context(), url.PathEscape(id)); err != nil {
```

Comment:
```suggestion
				if _, err := cli.api.ResourceServer.Read(cmd.Context(), id); err != nil {
```

No need to do `url.PathEscape`, this is already handled within the Go SDK.

Reply:
The same below.

Reply:
@sergiught You sure about this? Michael and I just went through this and IIRC, we still need this for both APIs and custom domains. Perusing the SDK repo and don't see that behavior.

Reply:
@willvedd it is done here: https://github.com/auth0/go-auth0/blob/bee2ceadcce411916cef360c448af604b4b17897/management/management_request.go#L29